### PR TITLE
Tags input

### PR DESCRIPTION
**Pull request by thecalcc**

### CODE REVIEW

File: app-typescript/components/TagInput.tsx
Comment (tomaskikutis) [inline app-typescript/components/TagInput.tsx]: don't use a default placeholder here - we can't translate strings in ui-framework project

File: examples/pages/react/TagInputDocs.tsx
Comment (tomaskikutis) [inline examples/pages/react/TagInputDocs.tsx]: don't commit commented out code :)

File: app/styles/form-elements/_tag-input.scss
Comment (tomaskikutis) [inline app/styles/form-elements/_tag-input.scss]: remove code instead of commenting it out

File: app/styles/primereact/_pr-tag-input.scss
Comment (tomaskikutis) [inline app/styles/primereact/_pr-tag-input.scss]: can you use more specific selectors so you don't need `!important`?

File: app-typescript/components/TagInput.tsx
Comment (dzonidoo) [reply app-typescript/components/TagInput.tsx]: we always use the default placeholder so i thought it was necessary now... I'll remove it now...

File: examples/pages/react/TagInputDocs.tsx
Comment (dzonidoo) [reply examples/pages/react/TagInputDocs.tsx]: agree :D

**Comment by thecalcc** [comment]
Old angular component isn't used anywhere so we're ready to merge if everything else is done.

**Review by tomaskikutis — APPROVED** [review]
fix tests before merging

**Pull request merged**
